### DISCUSSION TITLE
Suggestion: Shorter tooltip delays

### DISCUSSION THREAD

**Discussion by anderso** [jpmonettas/flow-storm-debugger#118] [Ideas]
Many of the buttons and tabs in flow-storm are non-obvious, which is fine for this kind of tool, and thankfully the tooltips are very helpful. I do find that they take a long time to display though, so I suggest a shorter delay.

The current delay is definitely longer than other apps I tried. I tried searching for best practices for delay, didn't find much.

**Comment by jpmonettas** [comment]
Make sense. This commit makes it 400ms instead of the default 1000ms https://github.com/flow-storm/flow-storm-debugger/commit/42ff6d9ee4c11b16cf9b57b1326dc816b4e824ec
